PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,12 @@ _Put an `x` in the boxes that apply. You can also fill these out after creating
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] I have added necessary documentation (if appropriate)
 
+## Testing instructions
+-Don't assume the tester knows the entire backstory of the issue, and don't force him/her to decipher the code to try and figure out what -it's doing or how to test it.
+-Do provide step by step instructions on how to test. 
+-Do note things to watch out for.
+-Do not what aspects the tester should try and break.
```

Comment:
Any chance this has a typo?

Reply:
no idea what this is

Reply:
Was it supposed to be "Do not do the aspects the tester should try and break."?

Reply:
Or was it "Do note* what aspects..."?

Reply:
Oh, i see now, that is a commit from Michael.

Disregard